website/docs/projects/m4c.md

```diff
@@ -0,0 +1,118 @@
+---
+id: m4c
+sidebar_label: M4C
+title: Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA
+---
+
+This project page shows how to use M4C model from the following paper, released under the MMF:
+
+- R. Hu, A. Singh, T. Darrell, M. Rohrbach, _Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA_. in CVPR, 2020 ([PDF](https://arxiv.org/pdf/1911.06258.pdf))
+
+```
+@inproceedings{hu2020iterative,
+  title={Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA},
+  author={Hu, Ronghang and Singh, Amanpreet and Darrell, Trevor and Rohrbach, Marcus},
+  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
+  year={2020}
+}
+```
+
+Project Page: http://ronghanghu.com/m4c
+
+## Installation
+
+Clone MMF, and build it with the following command.
+
+```
+cd ~/mmf
+python setup.py build develop
+```
```

Comment:
Since MMF is now fully merged into the Master branch, we can just ask users to follow the standard MMF installation procedure. So we can change the 6 lines above into:
```
Install MMF following the [installation guide](https://mmf.sh/docs/getting_started/installation/).
```